tasks/bump.rake

```diff
@@ -28,6 +28,7 @@ def bump_and_push(version)
   update_changelog(version)
   Bump::Bump.run(version, tag: true, commit: true, changelog: false)
 
+  sh %(git push)
```

Comment:
Minor fix, I somehow missed this when I was making the last changes; `git push --tags` apparently ONLY pushes tags, not the actual code.